apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx

```diff
@@ -85,28 +118,103 @@ describe("StopPageRedesign", () => {
     expect(mapPolylines).toHaveLength(10);
   });
 
+  const dateFormatter = (date: Date): string => {
+    return format(date, "yyyy-M-d HH:mm");
+  };
+
   it("should render alerts", () => {
-    const lowAlert: Alert = {
-      updated_at: "Updated: 4/11/2019 09:33A",
-      severity: 7,
-      priority: "low",
-      lifecycle: "upcoming",
-      active_period: [],
-      informed_entity: {} as InformedEntitySet,
-      id: "00005",
-      header: "There is construction at this station.",
-      effect: "other",
-      description: "",
-      url: "https://www.mbta.com"
-    };
+    const now = new Date();
+    const future1 = add(now, { days: 1 });
+    const lowAlerts: Alert[] = [
+      {
+        updated_at: "Updated: 4/11/2019 09:33A",
+        severity: 7,
+        priority: "low",
+        lifecycle: "new",
+        active_period: [[dateFormatter(now), dateFormatter(future1)]],
+        informed_entity: {} as InformedEntitySet,
+        id: "00005",
+        header: "There is construction at this station.",
+        effect: "other",
+        description: "",
+        url: "https://www.mbta.com"
+      }
+    ];
 
     jest
       .spyOn(useAlerts, "useAlertsByStop")
-      .mockImplementation(() => [lowAlert]);
+      .mockImplementation(() => lowAlerts);
 
     render(<StopPageRedesign stopId="123" />);
     expect(
       screen.queryByText("There is construction at this station.")
     ).not.toBeNull();
   });
+
+  it("should only render current alerts and not future alerts", () => {
```

Comment:
suggestion: Should there be a test for past alerts?

Reply:
Past alerts do not get handed to us from dotcom.  I guess if someone sits on a page long enough the current alert, could become a past alert.